db/migrate/20220915130804_change_column_name_key_long_read_qc_result.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Renamed column to avoid ambiguity
+class ChangeColumnNameKeyLongReadQcResult < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :long_read_qc_result, :key, :assay_type_key
```

Comment:
This tests: [spec/models/long_read_qc_result_spec.rb](https://github.com/sanger/unified_warehouse/pull/400/files#diff-9dcbe75aad0e75e0d16692ff6aefd446fe205be480ded782e5546eeb16f5597c) are out of date after this change on the field.